src/store.js

```diff
@@ -0,0 +1,100 @@
+// Store Class: Handles Storage
+class Store {
+  // Get Tasks : gets the tasks from store
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      // if there is nothing in storage
+      tasks = []; // create empty array
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks')); // convert them in array
+    }
+    return tasks;
+  }
+
+  // add the task to store
+  static addTask(task) {
+    const tasks = Store.getTasks(); // get tasks
+    tasks.push(task); // add new task
+    localStorage.setItem('tasks', JSON.stringify(tasks)); // convert them in string
+    // document.querySelector('.tasks-left-num').textContent = tasks.length;
```

Comment:
[Required] Kindly remove the unwanted comment in order to keep the code clean.

Reply:
thank you for the feedback @benshidanny11